spec/acceptance/foreman_rh_cloud_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper_acceptance'
+
+describe 'Scenario: install foreman with rh_cloud', if: os[:family] == 'redhat' do
```

Comment:
I still wonder if this should be `redhat` or a list? Look at the other code that is `['debian', 'ubuntu'].include?(os[:family])`.

Reply:
It gets executed on both CentOS Stream and Alma (I verified that), so 🤷‍♀️

Reply:
This is the reason we drink ;)